src/sentry/static/sentry/app/views/organizationEventsV2/eventDetails.jsx

```diff
@@ -67,7 +65,14 @@ class EventDetails extends AsyncComponent {
 
   handleClose = event => {
     event.preventDefault();
-    browserHistory.goBack();
+    const {location} = this.props;
+    // Remove modal related query parameters.
+    const query = omit(location.query, ['groupId', 'eventSlug', 'oldest']);
```

Comment:
Could "oldest" and "latest" just be handled as special cases of eventSlug values? This is similar how it's handled in the issue details views where "oldest" and "latest" are just special eventIds that are handled a bit differently. I think the nice thing about it is that we avoid two additional URL params.

Reply:
Sure. I like that idea. :+1: